common/interpolation/src/spline_interpolation.cpp

```diff
@@ -149,6 +153,80 @@ std::vector<double> getSplineInterpolatedValues(
 
   return res;
 }
+
+std::vector<double> getSplineInterpolatedDiffValues(
+  const std::vector<double> & base_keys, const std::vector<double> & query_keys,
+  const interpolation::MultiSplineCoef & multi_spline_coef)
```

Comment:
![image](https://user-images.githubusercontent.com/20228327/153788462-0252f823-57c4-4953-8527-b7b9cf8eae89.png)

Reply:
@TakaHoribe 
I added. Is it enough?
https://github.com/autowarefoundation/autoware.universe/pull/379/commits/71538785168f2230ea989545e7728f54344861d7

Reply:
thx!

Reply:
I recreated the commit since it was not with --signed-off
https://github.com/autowarefoundation/autoware.universe/pull/379/commits/d9653789ebb9a76b2686235f1bc2593d9e78e9b0